addons/event/report/event_event_templates.xml

```diff
@@ -218,6 +219,32 @@
     </t>
 </template>
 
+<!-- EVENT BADGE CARD (tool template used in all badge templates)-->
+
+<template id="event_report_template_badge_card">
+    <t t-set="badge_image_url" t-value="'data:image/png;base64,%s' % event.badge_image.decode('utf-8') if event.badge_image else ''"/>
+    <div class="o_event_badge_ticket_wrapper" t-attf-style="background-image: url({{badge_image_url}});">
+        <div class="position-relative h-100">
+            <h3 class="fw-bold text-center" t-field="event.name"/>
+            <div class="text-center mt-5 py-2">
+                <h2 class="mb-0" t-if="attendee" t-field="attendee.name"/>
+                <h2 class="mb-0" t-elif="not attendee"><span>John Doe</span> <span t-if="attendee_number" t-out="attendee_number + 1"/></h2>
+                <h4 t-if="attendee and attendee.company_id" class="o_event_badge_font_faded" t-field="attendee.company_id.name"/>
+                <h4 t-elif="not attendee"><span class="o_event_badge_font_faded">My Placeholder Company</span></h4>
+            </div>
+            <div class="position-absolute bottom-0 w-100 text-center">
+                <img t-if="event.organizer_id.image_256" class="o_event_badge_logo text-center" t-att-src="image_data_uri(event.organizer_id.image_256)"/>
+                <span t-if="event.badge_format != 'A4_french_fold'" class="o_event_badge_barcode_container"></span>
+                <t t-set="first_ticket" t-value="event.event_ticket_ids[0] if event.event_ticket_ids else None"/>
+                <t t-set="ticket" t-value="attendee.event_ticket_id if attendee else first_ticket if first_ticket else None"/>
+                <div class="text-center mt-2 w-100" t-attf-style="background-color: {{ticket.color if ticket and ticket.color else '#875A7B'}};">
+                    <div t-if="ticket" class="p-3 fs-3" t-out="ticket.name"/>
```

Comment:
shouldn't the `t-if="ticket"` be on the parent div ?

(when there's no ticket, badge on the right the div is not visible anyway)

![image](https://github.com/odoo/odoo/assets/54316226/82913329-69ed-47a1-aca2-87b0a305a828)